wand-library-in-python/index.md

```diff
@@ -0,0 +1,182 @@
+### Wand Library in Python
+[ImageMagick](https://imagemagick.org/index.php) is an attribute developed to change the format of an image. ImageMagick is extensively used due to its capability to work with image formats in the broader scope and its precision and ease of use. 
+
+Therefore, we will take a deep look at the wand Python binding, which Imagemagick developed in this article. Furthermore, with the help of several examples, which will well explain, we will learn about the features and applications of the Wand library in the Python programming language.
+### Defining Wand library.
+Using Python programming language, which created a library known as Imagemagick, and this is what we term as the Wand. The library, therefore, provides Imagick API functionalities across various versions of Python. We can use the Wand to edit and open images, but in collaboration with NumPy, it offers significant functions for machine learning codes. Below are some of its uses.
```

Comment:
`provides Imagick API functionalities across various versions of Python`

What functionalities